### PR TITLE
FEATURE: BEND-14 Get Photos Endpoint

### PR DESCRIPTION
**Pull request by octapf**

### CODE REVIEW

**Comment by octapf** [comment]
@whitehatdevv I don't understand why it did all this commits before i assigned the labels to the pull request. Can you explain me?

image

**Comment by octapf** [comment]
> > @whitehatdevv I don't understand why it did all this commits before i assigned the labels to the pull request. Can you explain me?
> > image
> 
> Because you had created the PR to master not to develop ;). Please be careful, I've just changed it Captura de pantalla 2022-01-20 a las 8 02 41

Oh , ok! I'll be more careful next time. Thanks!

**Comment by whitehatdevv** [comment]
You have the conflicts yet

**Comment by octapf** [comment]
> You have the conflicts yet

Resolved!

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
...
 
+<<<<<<< HEAD
 Micro services neeaded tso perform all authentication actions
+=======
+<<<<<<< HEAD
+Micro services neeaded tso perform all authentication actions
+=======
+Micro servicse neeaded tso perform all authentication actions
+>>>>>>> f9aece599735b6ecd5b8cb28adaa10aba87c8d26
+>>>>>>> develop
```
Comment (whitehatdevv) [inline README.md]: You MUST read the content you are uploading to GITHUB, this file is wrong, it has CONFLICTS, I will fix it on a new commit, please be careful with that, you can destroy the project

Comment (octapf) [reply README.md]: Sorry is just that i had a massive error cause by Git Lens which nobody new how to resolve it in many forums. So i had to reinstall vscode and it got fixed. But at the beginning it didn't how me chance to the controller to save incoming changes, and i was trying to solve the problem with the readme.md file that it won't cause many troubles.
I have reinstalled vscode and now is working properly.

**Pull request merged**